src/ucp/tag/rndv.c

```diff
@@ -596,6 +599,236 @@ static void ucp_rndv_req_send_rma_get(ucp_request_t *rndv_req, ucp_request_t *rr
     ucp_request_send(rndv_req, 0);
 }
 
+UCS_PROFILE_FUNC_VOID(ucp_rndv_recv_frag_put_completion, (self, status),
+                      uct_completion_t *self, ucs_status_t status)
+{
+    ucp_request_t *freq     = ucs_container_of(self, ucp_request_t,
+                                               send.state.uct_comp);
+    ucp_request_t *req      = freq->send.rndv_put.sreq;
+    ucp_request_t *rndv_req = (ucp_request_t *)freq->send.rndv_put.remote_request;
```

Comment:
we don't add whitespace between the name of the type and `*` when we do casting:
`(ucp_request_t *)` -> `(ucp_request_t*)`